tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "commonjs",
+    "lib": ["ES2020"],
+    "strict": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "resolveJsonModule": true
+  },
+  "references": [
+    { "path": "./shared/commons" },
+    { "path": "./components/analytics" },
```

Comment:
Could it be `{ "path": "./components/*" }` to avoid adding a new reference for each component? Or is it intentional for more control?

Reply:
It's sadly not possible, TS allows only complete paths for it to work.